torch_np/_ndarray.py

```diff
@@ -58,6 +107,17 @@ def strides(self):
     def base(self):
         return self._base
 
+    @property
+    def flags(self):
+        # Note contiguous in torch is assumed C-style
+        flag_to_value = {"C_CONTIGUOUS": self._tensor.is_contiguous()}
+        if flag_to_value["C_CONTIGUOUS"]:
+            # There's no proper way to determine if a tensor is Fortran-style
+            # contiguous in torch, but at least we know it isn't when it is
+            # C-style.
```

Comment:
Strictly speaking, 1D arrays can be both C and F contiguous. Not sure how deep into that rabbit hole we need fall though.

Reply:
Oh right—this was a bit magical to begin with so removed.